### PR TITLE
Integrate existing tests

### PR DESCRIPTION
**Pull request by shikamu**
finished integrating the last test

### CODE REVIEW

File: tests/backend/test_headers.py [hunk 1]
```diff
@@ -12,3 +13,18 @@ def test_ip_resolver_not_returned(graphql_endpoint, query_me):
...
+
+def test_security_headers_need_highest_possible_grade(domain):
+    res = requests.get('https://securityheaders.com/?q=' +
+                       domain + '&followRedirects=on')
+    assert res.headers['X-Grade'] == 'A+'
+
+
+def test_ssllabs_scan_grade_is_highest(domain):
+    cmd = sh.Command('../../ssllabs-scan')
+    result = cmd('-grade', '-json-flat', '-verbosity', 'error', domain)
```
Comment (zadigus) [inline tests/backend/test_headers.py]: is it really necessary to have all those args here? isn't 
```
result = cmd('-grade', domain)
```
enough?

**Review by zadigus — CHANGES_REQUESTED** [review]

File: tests/backend/test_headers.py (hunk 1 above)
Comment (shikamu) [reply tests/backend/test_headers.py]: maybe, need to try it. will check later

**Comment by zadigus** [comment]
https://github.com/shopozor/security-tests/issues/24

**Comment by zadigus** [comment]
https://github.com/shopozor/security-tests/issues/25

**Review by zadigus — APPROVED** [review]
ok

**Pull request merged**